community/shell/src/main/java/org/neo4j/shell/impl/JLineConsole.java

```diff
@@ -40,7 +40,7 @@ static JLineConsole newConsoleOrNullIfNotFound( ShellClient client )
 				Class.forName( "jline.console.ConsoleReader" ).newInstance();
 			consoleReader.getClass().getMethod( "setBellEnabled",
 				Boolean.TYPE ).invoke( consoleReader, false );
-			consoleReader.getClass().getMethod( "setBellEnabled", Boolean.TYPE ).invoke(
+			consoleReader.getClass().getMethod( "setExpandEvents", Boolean.TYPE ).invoke(
```

Comment:
if you check the original code, you see that setBellEnabled was in there twice. That's why the change shows up as changing the 2nd repetition of setBellEnabled.